app/jobs/regular/enable_bootstrap_mode.rb

```diff
@@ -7,6 +7,7 @@ class EnableBootstrapMode < ::Jobs::Base
     def execute(args)
       raise Discourse::InvalidParameters.new(:user_id) unless args[:user_id].present?
       return if SiteSetting.bootstrap_mode_enabled
+      return if !SiteSetting.show_create_topics_notice
```

Comment:
I'm actually think that it might be best to create a new SiteSetting here instead of piggy backing on the `show_create_topics_notice` setting that could confuse someone in the future.

Maybe `SiteSetting.disable_enable_bootstrap_mode` lol